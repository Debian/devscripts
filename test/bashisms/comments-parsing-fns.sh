#!/bin/sh

echo $# ; echo -e BASHISM
