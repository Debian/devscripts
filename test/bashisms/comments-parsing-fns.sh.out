possible bashism in bashisms/comments-parsing-fns.sh line 3 (echo -e):
echo $# ; echo -e BASHISM
